contracts/test/ContractRegistry.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.5.0;
+
+import "contract-registry/contracts/interfaces/IContractRegistry.sol";
```

Comment:
This should be in a different folder because it's not a test (e.g. `contracts/stubs` or `contracts/test/stubs/`